src/Drush/Commands/PlaywrightDrushCommands.php

```diff
@@ -251,11 +253,16 @@ public function getNodePathWithTitle(string $title): string {
    * @command test:node-get-path-alias
    * @aliases ngpath-alias
    */
-  public function getNodePathAliasWithTitle(string $title): string {
+  public function getNodePathAliasWithTitle(string $title, $langcode = ''): string {
     $nodes = $this->getEntityTypeManager()->getStorage('node')->loadByProperties([
       'title' => $title,
     ]);
     if ($node = reset($nodes)) {
+      if ($langcode) {
+        return $this->getAliasManager()->getAliasByPath('/node/' . $node->id(), $langcode);
+      }
+      // Just a precaution in case getAliasByPath() changes its signature: do
+      // not pass langcode in the default case.
```

Comment:
Instead of this extra if, I could also have replaced the single line below here by
`return $this->getAliasManager()->getAliasByPath('/node/' . $node->id(), $langcode ?: NULL);`
but somehow it felt like Drupal may at some point start preventing explicitly passing a NULL.

I really don't care one way or the other.